.github/workflows/build-and-release.yml

```diff
@@ -70,7 +70,7 @@ jobs:
         run: |
           echo CONTENTFUL_SPACE="${{ secrets.CONTENTFUL_SPACE }}" >> .env
           echo CONTENTFUL_ACCESS_TOKEN="${{ secrets.CONTENTFUL_ACCESS_TOKEN }}" >> .env
-          echo GITHUB_TOKEN="${{ secrets.GITHUB_TOKEN }}" >> .env
+          echo GITHUB_TOKEN="${{ secrets.CMS_REPO_TOKEN }}" >> .env
```

Comment:
You'll need to change `echo GITHUB_TOKEN` to `echo CMS_REPO_TOKEN` as well

Reply:
That might help! Thanks!